shared/api/functions.js

```diff
@@ -117,7 +117,11 @@ module.exports = {
       .catch((err) => {
         // toleration on 4XX errors because on some status, for exemple deleting the API
         // will return a 404 err code if item has been deleted.
-        if (err.response.status >= 500) {
+        if (err.response === undefined) {
```

Comment:
For reviewers: there was a bug in this function. Before my change, it was only used once (in deleteFunctionsByIds), and I'm pretty sure it was never called anyway because the logic to delete functions when they don't exist anymore in the serverless.yml file does not work 🤷

Anyway, now that I'm using this method elsewhere, better fix it.